go-controller/pkg/config/config.go

```diff
@@ -58,18 +58,19 @@ var (
 
 	// Default holds parsed config file parameters and command-line overrides
 	Default = DefaultConfig{
-		MTU:                   1400,
-		ConntrackZone:         64000,
-		EncapType:             "geneve",
-		EncapIP:               "",
-		EncapPort:             DefaultEncapPort,
-		InactivityProbe:       100000, // in Milliseconds
-		OpenFlowProbe:         180,    // in Seconds
-		OfctrlWaitBeforeClear: 0,      // in Milliseconds
-		MonitorAll:            true,
-		LFlowCacheEnable:      true,
-		RawClusterSubnets:     "10.128.0.0/14/23",
-		Zone:                  types.OvnDefaultZone,
+		MTU:                    1400,
+		ConntrackZone:          64000,
+		EncapType:              "geneve",
+		EncapIP:                "",
+		EncapPort:              DefaultEncapPort,
+		InactivityProbe:        100000, // in Milliseconds
+		OpenFlowProbe:          180,    // in Seconds
+		OfctrlWaitBeforeClear:  0,      // in Milliseconds
+		MonitorAll:             true,
+		LFlowCacheEnable:       true,
+		RawClusterSubnets:      "10.128.0.0/14/23",
+		Zone:                   types.OvnDefaultZone,
+		MaxUserDefinedNetworks: 30000,
```

Comment:
I am guessing this includes primary and secondary right?

Reply:
cc @martinkennelly

Reply:
> I am guessing this includes primary and secondary right?

I think is just primary since there is no egress for secondary, Do I change the name to `MaxUserDefinedPrimaryNetworks` ?

Reply:
I think we plan to support pod2Egress on secondary in the future even if we don't support them today. I'd keep this for both primary and secondary so keeping it UDN is fine, but good to leave a comment outlining that.